src/Message.chpl

```diff
@@ -344,6 +344,14 @@ module Message {
             }
             return val_list;
         }
+
+        /*
+        Return bool if param_list contains given key name
+        */
+        proc contains(key: string): bool {
```

Comment:
Noting for myself that it may be worth updating all of this functionality to use `List` objects instead of arrays since we moved to that for the main args.

Reply:
Noting for you #1903

Reply:
Thanks!